### ISSUE TITLE
PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted

### ISSUE THREAD

**Issue by ryanlabelle** [themeum/kirki#1404]
### Issue description:
When using the Number control I get a server 500. This happens when I go up or down.

PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 32 bytes) in plugins/kirki/field/class-kirki-field-number.php on line 69, referer: wp-admin/customize.php?return=%2Fembark%2Fwp-admin%2Fplugins.php&changeset_uuid=98e6aa69-3c26-4f60-a360-a32ff2a83e5d

### Version used:
3.0.3

### Code to reproduce the issue (config + field(s))
```
Embark_Kirki::add_field( 'embark_theme', array(
    'type'        => 'number',
    'settings'    => 'themo_nav_top_margin',
    'label'       => esc_html__( 'Navigation Top Margin', 'embark' ),
    'description' => esc_html__( 'Set top margin value for the navigation bar', 'embark' ),
    'section'     => 'menu',
    'default'     => 19,
    'choices'     => array(
        'min'  => '0',
        'max'  => '300',
        'step' => '1',
    ),
    'output' => array(
        array(
            'element'  => '.navbar .navbar-nav',
            'property' => 'margin-top',
            'units'    => 'px',
        ),
        array(
            'element'  => '.navbar .navbar-toggle',
            'property' => 'top',
            'units'    => 'px',
        ),
        array(
            'element'  => '.themo_cart_icon',
            'property' => 'margin-top',
            'value_pattern' => 'calc($px + 12px)'
        ),
    ),
) );
```

**Comment by aristath** [comment]
That's weird...

That last commit (https://github.com/aristath/kirki/commit/16c1d05fa9e512eddf1a9e4a1aafc3189fa7eee6) should fix your issue, can you please test it?

**Comment by ryanlabelle** [comment]
That worked! Thanks!!

**Comment by aristath** [comment]
Better fix for this was added in https://github.com/aristath/kirki/commit/cdccd553c05fc22b1750d40b551f385b1e0206c6
Will be included in 3.0.4 later today.
